src/errors/BaseError.ts

```diff
@@ -13,7 +13,7 @@ export default class BaseError extends Error {
     protected properties: ErrorProperties;
     hasBeenLogged: boolean = false;
 
-    constructor(message: string, exception?: Record<string, unknown>) {
+    constructor({ message, exception }: BaseErrorArgs) {
```

Comment:
Nå som dette skal bli brukt av andre også, så lurer jeg på om exception heller burde vært innerException, med Get. Slik det blir enklere fra utsiden å få tak i denne og behandle den.
Vi må bare huske å samtidig sjekke/forandre application insights logging. Og også passe på at logError skriver ut denne.

Reply:
Godt poeng, det kunne være sin egen task eller US kanskje?

Reply:
Slik jeg tolker det, så bør det gjøres her. Det er det denne US går ut på, å gjøre error håndtering tilgjengelig fra core for alle applikasjoner.

Reply:
Det bør være en egen task ja, men del av denne US :)